src/forms-opportunities/utils.js

```diff
@@ -211,10 +213,13 @@ async function convertToOpportunityData(opportunityType, metricObject, context)
   let opportunityData = {};
 
   if (opportunityType === FORM_OPPORTUNITY_TYPES.LOW_CONVERSION) {
+    log.info(`low conversion metric object : ${JSON.stringify(metricObject)}`);
```

Comment:
I think we should remove these logs in favor in reducing log volume.
we are already logging the oppoty data atleast once.